### ISSUE TITLE
plugin refresh after install

### ISSUE THREAD

**Issue by yellowtide** [Sleitnick/AeroGameFramework#12]
after installing the framework via require(id)(), the plugin needs to be manually refreshed to start using it

**Comment by Sleitnick** [comment]
Good catch. It's supposed to do this, so there's something going wrong. I'll work on a fix for this.
